### PR TITLE
-Changed CompatibilityChecker.cs to allow Kerbal Space Program Versio…

### PR DESCRIPTION
**Pull request by PhilG0lding**
For KSP version 1.2.9

-Updated CompatibilityChecker.cs and DisclaimerWindow.cs to fix build errors related to PopupDialog (invalid overload?)

### CODE REVIEW

File: Client/DisclaimerWindow.cs [hunk 1]
```diff
@@ -20,7 +20,8 @@ public static void SpawnDialog()
 
             PopupDialog.SpawnPopupDialog(new Vector2(0, 0),
                 new Vector2(0, 0),
-                new MultiOptionDialog(disclaimerText,
+                new MultiOptionDialog("Disclaimer",
```
Comment (MattRamsay) [inline Client/DisclaimerWindow.cs]: :champagne:

File: Client/Utilities/CompatibilityChecker.cs [hunk 2]
```diff
@@ -50,7 +50,7 @@ public static bool IsCompatible()
         {
             const int compatibleMajor = 1;
             const int compatibleMinor = 2;
-            const int compatibleRevision = 2;
+            const int compatibleRevision = 9;
```
Comment (MattRamsay) [inline Client/Utilities/CompatibilityChecker.cs]: Wicked!

**Comment by RockyTV** [comment]
The reason why we haven't changed to 1.2.9 is because 1.2.9 is a pre-release version. Anyway, why the changes to the dialogs?

**Comment by PhilG0lding** [comment]
I created this as a group of people were hoping to use it on pre-release (And then on release). I was hoping that it would be added to a new branch, perhaps "Pre-release" and people can put similar changes through in future.

Changes to dialogs, looks like Assembly-CSharp.dll now requires a Dialog name in the constructor parameters, which I've added.

**Comment by godarklight** [comment]
I've no problem with supporting this, as it will almost certainly require no changes to 1.3.0, and it is available publicly.

My big question is where do these KSP versions keep coming from with no dev team? :P

**Comment by RockyTV** [comment]
Fixed via ac91be7b647adaaf8f64d093abd043cbdcb8dc3e

**Pull request closed without merging**